lib/app/ui.nit

```diff
@@ -211,7 +230,7 @@ abstract class TextView
 	# depending on the customization options of each platform.
 	# For consistent results, it is recommended to use only on instances
 	# of `Label` and `size` should be either 0.0, 0.5 or 1.0.
-	fun align=(center: nullable Float) is autoinit do end
+	fun align=(align: nullable Float) is autoinit do end
```

Comment:
Once again, this one is a simple fix which is a no-brainer, it's not worth a dedicated PR.